member_directory/members.json

```diff
@@ -0,0 +1,103 @@
+{
+    "members": [
+        {
+            "avatarImg": "",
```

Comment:
The avatar images aren't rendering, because the value for `avatarImg` is an empty string for each object in the `members` array.

You're using the value for `avatarImg` in the for loop here:

```ts
<img
  :src="member.avatarImg"
  class="member-image object-cover rounded-full w- 
  16 h-16"
/>
```

The problem is that the function you wrote to map the paths for the images doesn't execute on mount. I think we could simplify things by just putting the full paths in the json since it's a separate file.

That will clean up the vue file even more.

<img width="736" alt="Screenshot 2024-03-12 at 9 10 08 PM" src="https://github.com/ourjapanlife/findadoc-web/assets/31802656/88f56441-2b35-440a-9893-8f8ffe914b73">

Reply:
I changed the files to be in the json file and got rid of the function as suggested

Reply:
Comment below describes what I did to fix it